spec/jobs/reports_uploader_job_spec.rb

```diff
@@ -4,6 +4,13 @@
   let(:report_uploader) { described_class.new }
   subject { report_uploader.perform }
 
+  describe '#expiration' do
+    subject(:expiration) { report_uploader.expiration }
+    it 'returns 24' do
```

Comment:
might be clearer to say `returns 24 hours`

Reply:
I thought it did, I must have CMD+Z'd too much! :D 
Updated